### PR TITLE
Update drafttimings.

### PR DESCRIPTION
**Pull request by bobsaget4lyfe**

### CODE REVIEW

**Comment by bobsaget4lyfe** [comment]
Wont have broadband until wednesday so cannot test until then.

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (howardchung) [inline .idea/vcs.xml]: remove this file?

**Comment by bobsaget4lyfe** [comment]
My my do I not understand git

**Comment by howardchung** [comment]
try `git checkout package-lock.json` and that should remove the changes

File: processors/processDraftTimings.js [hunk 2]
```diff
@@ -4,87 +4,61 @@
...
-        }
-      } */
-      let ind2;
-      // find the time of the end of the previous order
-      for (let i = 0; i < draftTimings.length; i += 1) {
-        const currpick = draftTimings[i];
-        if (currpick.order === (pnb.order - 1)) {
-          ind2 = i;
+  // ignore Source 1 games
+  if (typeof draftTimings[0] !== 'undefined') {
```
Comment (howardchung) [inline processors/processDraftTimings.js]: `!draftTimings.length`?

**Comment by howardchung** [comment]
seems to be a test failure:
https://travis-ci.org/odota/core/builds/345562638?utm_source=github_status&utm_medium=notification

**Comment by bobsaget4lyfe** [comment]
Mmm i wonder why it failed the first time.

**Pull request merged**